### ISSUE TITLE
PutioAPI.Files.Upload 401 but PutioAPI.Account.Info() works

### ISSUE THREAD

**Issue by ptheofan** [putdotio/putio.js#61]
Hi, I do not understand what I am doing wrong. I initialize the API client like this

```ts
      const putioConfig = this.configService.get('putio');
      this.api = new PutioAPI({
        clientID: putioConfig.clientId,
      });
      this.api.setToken(putioConfig.oAuthToken);
```

with this API when I call the `Account.Info()` it works perfectly as expected.

```ts
  const accountInfo = await putio.Account.Info();
  this.logger.log(`Uploading as ${accountInfo.data.info.username}`);
```

However when I call the `Files.Upload(...)` I get 401

```
      await api.Files.Upload({
        file: absolutePath,
        fileName: filename,
        parentId: targetFolderId,
      });
```

is there some issue with the `sdk` or am I doing something wrong?

**Comment by altaywtf** [comment]
hey @ptheofan it's on me... 😞 working on a fix now!

**Comment by altaywtf** [comment]
should be fixed in `v8.35.0`, sorry for the late response...
